classes/cave.js

```diff
@@ -81,6 +85,8 @@ class Cave {
          */
         this.adminEmojis = new Discord.Collection();
```

Comment:
Do we need this collection? I will check and see, I don't think we need it if we will do the change above to use an object to hold the emojis.

Reply:
I kept it because it contains only the emojis on the admin console and the code loops through it a couple times.